src/js/components/covid19/OpportunityProjectPage.jsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * OpportunityProjectPage.jsx
+ * Created by Brett Varney 10/27/21
+ */
+
+import React from 'react';
+import PageWrapper from 'components/sharedComponents/PageWrapper';
+import { topPageMetaTags } from 'helpers/metaTagHelper';
+import { FlexGridContainer, FlexGridRow, FlexGridCol } from 'data-transparency-ui';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useDispatch } from 'react-redux';
+import { showModal } from 'redux/actions/modal/modalActions';
+
+require('pages/covid19/opportunityProject.scss');
+
+const OpportunityProjectPage = () => {
+    const dispatch = useDispatch();
+    const handleClick = (e) => {
+        e.preventDefault();
+        dispatch(showModal(e.target.href));
+    };
+
+    const externalLink = (href, linkText) => (
```

Comment:
We have an existing shared component, `components/sharedComponents/ExternalLink.jsx`

Reply:
It can't render as inline text, only as a button, which is a 508 violation here.

Reply:
Sounds like this is a good opportunity to extend the existing component to support links in addition to buttons!